docker-storage-setup.sh

```diff
@@ -940,37 +937,37 @@ setup_docker_root_volume() {
     Fatal "DOCKER_ROOT_VOLUME_SIZE value $DOCKER_ROOT_VOLUME_SIZE is invalid."
   fi
 
-  if ! create_lv $DOCKER_ROOT_VOLUME_SIZE $DOCKER_ROOT_LV_NAME; then
-    Fatal "Failed to create volume $DOCKER_ROOT_VOLUME_SIZE of size ${DOCKER_ROOT_VOLUME_SIZE}."
+  if ! create_lv $DOCKER_ROOT_VOLUME_SIZE $ROOT_LV_NAME; then
+    Fatal "Failed to create volume $ROOT_LV_NAME of size ${DOCKER_ROOT_VOLUME_SIZE}."
   fi
 
-  if ! mkfs -t xfs /dev/$VG/$DOCKER_ROOT_LV_NAME > /dev/null; then
-    Fatal "Failed to create filesystem on /dev/$VG/${DOCKER_ROOT_LV_NAME}."
+  if ! mkfs -t xfs /dev/$VG/$ROOT_LV_NAME > /dev/null; then
+    Fatal "Failed to create filesystem on /dev/$VG/${ROOT_LV_NAME}."
   fi
 
-  if ! mount_docker_root_volume; then
-    Fatal "Failed to mount docker root volume ${DOCKER_ROOT_LV_NAME} on ${DOCKER_ROOT_DIR}"
+  if ! mount_root_volume; then
+    Fatal "Failed to mount container runtime root volume ${ROOT_LV_NAME} on ${ROOT_VOLUME_MOUNT_DIR}"
```

Comment:
This was a review comment from @rhatdan 
https://github.com/projectatomic/docker-storage-setup/pull/181#discussion_r95842498

Dan, WDYT ?